.gitignore

```diff
@@ -1,3 +1,4 @@
+# Android Studio & IntelliJ project files
```

Comment:
I still have to set up your Android studio to ignore certain file but remember only select the files that you know you made changes to when committing and pushing to the repo.

Reply:
Okay, thanks!